CHANGELOG.md

```diff
@@ -10,6 +10,22 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+### Changed
+
+### Deprecated
+
+### Removed
+
+- Removed metrics test package `go.opentelemetry.io/otel/sdk/export/metric/metrictest`. (#2105)
```

Comment:
_Sorry if it is a stupid question._

Why this is in `Unreleased`? Do you plan to release this package using a commit before it was removed?

Reply:
that change was to a module in the `experimental metrics` version set that is not getting a new tag at this time, so the change is effectively unreleased.

Reply:
That information is contained in [`versions.yaml`](https://github.com/open-telemetry/opentelemetry-go/pull/2133/files#diff-1c4cd801df522f4a92edbfb0fea95364ed074a391ea47c284ddc078f512f7b6a).